src/Controllers.f90

```diff
@@ -223,6 +223,8 @@ SUBROUTINE YawRateControl(avrSWAP, CntrPar, LocalVar, objInst)
         REAL(8), SAVE :: Y_Err                                  ! Yaw error (deg)
         REAL(8)       :: YawRateCom                             ! Commanded yaw rate
         REAL(8)       :: deadband                               ! Allowable yaw error deadband (rad)
+        REAL(8)       :: Time                                   ! Current time
```

Comment:
The time is available in LocalVar%Time - probably should use that one for consistency